R/rd-family.R

```diff
@@ -1,3 +1,15 @@
+topics_process_family_prefix <- function(family) {
+
+  # read custom family prefix from roxygen meta
+  opts <- roxy_meta_read("family.prefix")
```

Comment:
To avoid future namespace clashes, I think this should be `rd_family_prefix`. 

But I'm not sure `prefix` is the right word? Maybe `title` would be better since you're looking up the title from the prefix?